README.md

```diff
@@ -15,22 +15,72 @@ It's based on [MinIO official image](https://hub.docker.com/r/minio/minio) and [
 
 ## Configuration
 
-### MinIO bucket
+### MinIO console
 
 Login to MinIO console `https://<project>.ddev.site:9090` login with credentials `ddevminio:ddevminio` and create a bucket.
 
-### Composer package
+### File access
 
-Your project will most likely require the [AWS PHP SDK](https://packagist.org/packages/aws/aws-sdk-php). You can install it like this
+Project docker instances can access MinIO api via `http://minio:10101`
+
+DDEV Router is configured to proxy the requests to `https://<project>.ddev.site:10101` to MinIO S3 Api.
+
+Example URLs for accessing files are
+
+| Bucket   | File path              | Internal URL                                     | External URL                                                    |
+|----------|------------------------|--------------------------------------------------|-----------------------------------------------------------------|
+| `photos` | `vacation/seaside.jpg` | `http://minio:10101/photos/vacation/seaside.jpg` | `https://<project>.ddev.site:10101/photos/vacation/seaside.jpg` |
+| `music`  | `tron/derezzed.mp3`    | `http://minio:10101/music/tron/derezzed.mp3`     | `https://<project>.ddev.site:10101/music/tron/derezzed.mp3`     |
+
+## Connecting from PHP
+
+### Installation
+
+Since MinIO is S3 compatible you can use [AWS PHP SDK](https://packagist.org/packages/aws/aws-sdk-php). Install it with composer:
 
 ```bash
 $ ddev composer require aws/aws-sdk-php
 ```
-Project docker instances can access MinIO api via `http://minio:10101`
 
-### Public access
+### Basic usage
```

Comment:
This is great stuff. If there's not already a test that does exactly this, please consider adding one.